_episodes/05-preprocessor.md

```diff
@@ -314,11 +341,8 @@ simple preprocessor and diagnostic setup for that:
 >         preprocessor: prep_regrid
 >         mip: Amon
 >         grid: gn  # can change for variables from the same model
->         start_year: 1970  # some 30 year period
+>         start_year: 1979
 >         end_year: 2000
->         additional_datasets:
```

Comment:
Removal of observation data set should also be reflected in text. Now says we use observations in 2 vars. While with this change only pr has observations.

Reply:
it is fixed now.